CHANGELOG.md

```diff
@@ -2,7 +2,25 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.16.0] - 2017-01-29
+## [0.16.0] - 2018-04-03
+### Added
+- Add Latency to WebUI topology links
+  Packet Injector now allows to increment ICMP `id` for each packet
+  New `Light` WebUI theme
+  Add `Has` Gremlin step to `SocketInfo` step allowing to filter socket information
```

Comment:
- New `socketinfo` probe to retrieve active sockets of a host. The new `Sockets` Gremlin step can be used to retrieve socket information corresponding to flows.